k-distribution/tests/regression-new/markdownSelectors/Makefile

```diff
@@ -2,7 +2,7 @@ DEF=test
 EXT=test
 TESTDIR=.
 KOMPILE_BACKEND=haskell
-KOMPILE_FLAGS=--md-selector 'k|(keep&!discard)'
+KOMPILE_FLAGS=--md-selector '(k|keep) & !discard'
```

Comment:
Can you please explain what this does? It keeps only those for tests that are marked either k or keep and are not  discard? I.e., the discards pass through the parser but are not compiled?

Reply:
The `--md-selector` flag consumes a boolean expression over tag names, and keeps only those code blocks that satisfy the expression. Non-satisfying blocks are discarded before being parsed. https://kframework.org/k-distribution/k-tutorial/1_basic/08_literate_programming/